tests/test_stability_funs.py

```diff
@@ -522,6 +523,40 @@ def test_ballooning_geometry(tmpdir_factory):
         np.testing.assert_allclose(sqrt_g_PEST, 1 / (B_sup_zeta / psi_r))
 
 
+@pytest.mark.unit
+def test_grad_alpha_zeta0_maps():
+    """Test computation of gds2."""
+    eq = desc.examples.get("W7-X")
+    iota_zeta0 = np.linspace(-np.pi / 2, np.pi / 2, 15)[:, np.newaxis]
+    data = eq.compute(
+        ["alpha_r (secular)", "iota_r", "iota", "gds2", "c ballooning"],
+        zeta0=iota_zeta0,
+    )
+    gds2 = data["gds2"]
+    c_ballooning = data["c ballooning"]
+    data = eq.compute(
```

Comment:
@rahulgaur104 This line redefines $\alpha_{DESC}$ to be the same $\alpha$ in your ballooning paper ($\alpha_{DESC} + \iota \zeta_0$). Therefore statements like ``np.testing.assert_allclose(gds2, data["g^aa"] * data["rho"] ** 2)`` must pass if `gds2` is computed correctly.

Reply:
It will fail on `master` because it tests that `gds2` $=$ ρ² |∇(α + ι ζ₀)|². This relation is not true with the introduction of the sign factors: sign(iota) on the `g^ra` term.